scripts/camViewer

```diff
@@ -148,26 +148,27 @@ usage: $0 options
 start the viewer for controls cameras
 
 OPTIONS:
--c:            camera name as in camera list or gige #
--m:            bring up the edm screen
--r:            reboot the IOC
--l:            print list of cameras
--w #:         (wait for # hours to ask to renew, default 2 and 12 for GIGEs)
--u #:          update rate limit (default 5)
--H hutch:      use a specific hutches camviewer config file
--e|--enable:   enable camera ioc
--d|--disable:  disable camera ioc
+-c|--cam    :   camera name as in camera list or gige #
+-m|--main   :   bring up the edm screen
+-r|--reboot :   reboot the IOC
+-l|--list   :   print list of cameras
+-w|--wait # :   (wait for # hours to ask to renew, default 2 and 12 for GIGEs)
+-u|--rate # :   update rate limit (default 5)
+-H|--hutch  :   use a specific hutches camviewer config file
```

Comment:
```suggestion
-c|--cam    :   camera name as in camera list or gige #
-m|--main   :   bring up the edm screen
-r|--reboot :   reboot the IOC
-l|--list   :   print list of cameras
-w|--wait   :   (wait for # hours to ask to renew, default 2 and 12 for GIGEs)
-u|--rate   :   update rate limit (default 5)
-H|--hutch  :   use a specific hutches camviewer config file
```
for consistency. But probably clearer to add arguments after all options that require values instead.